README.md

```diff
@@ -0,0 +1,74 @@
+
+
+<div style="display: flex; justify-content: center; margin-top: 10px">
+  <img src="https://cdt-art-ai.ac.uk/app/themes/stirtingale/assets/img/logo.svg" alt="" style="width: 200px;">
+</div>
+
+# Medical Diagnosis Game
+
+Medical Diagnosis Game (MDG) with AI advice for individuals 
+
+Python code for research into how AI can affect multidisciplinary team decision-making within the context of healthcare. 
+
+## Application
+- The package is designed to be used with Psychopy & Python 2.7.18 
+
+## Running MDG
+
+Please use Python 2.7.18 to run MDG. This can be downloaded from
+https://www.python.org/downloads/release/python-2718/
+
+Then run the following commands:
+
+```bash
+pip install -r ../requirements.txt
+python MDG_Spring2020_1.0_AIAdvice.py
+```
+
+
+## Install Local Development Setup
+
+If you wish to develop MDG locally, the easiest way is to use the following steps:
+
+- clone this repo
+  
+```bash
+git clone https://github.com/ec531/MDG.git
+```
+
+- copy example.env to .env and fill out your user details 
+
+```bash
+cp template.env .env
+```
+
+- Move a version of the @todo.
+```bash
+cp src/@todo
+```
+
+- install the python required packages:
+```bash
+pip install -r ../requirements.txt
+```
+
+## Usage
+- Run python MDG_Spring2020_1.0_AIAdvice.py
+
+## Contributing
+- no contributions accepted at this time
+
+## License
+- no license at this time
+
+## Acknowledgements
+- @todo
+
+## References
+- [Psychopy]
```

Comment:
Peirce, JW (2007) PsychoPy - Psychophysics software in Python. Journal of Neuroscience Methods, 162(1-2), 8-13.
  Peirce, JW (2009) Generating stimuli for neuroscience using PsychoPy. Frontiers in Neuroinformatics, 2:10. doi: 10.3389/neuro.11.010.2008